terminate called after throwing an instance of 'std::runtime_error'
  what():  No filter rule for system call: kill
Aborted (core dumped)
